cmd/commands/subvolume.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"github.com/rook/kubectl-rook-ceph/pkg/subvolume"
+	"github.com/spf13/cobra"
+)
+
+var SubvolumeCmd = &cobra.Command{
+	Use:   "subvolume",
+	Short: "manages the stale-subvolumes",
+	Args:  cobra.ExactArgs(1),
+}
+
+var listCmd = &cobra.Command{
+	Use:   "ls",
+	Short: "Print the list of stale subvolumes no longer in use.",
+	Args:  cobra.MaximumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		clientsets := GetClientsets(cmd.Context())
+		VerifyOperatorPodIsRunning(cmd.Context(), clientsets, OperatorNamespace, CephClusterNamespace)
+		staleSubvol := cmd.Flag("type").Value.String()
+		subvolume.List(cmd.Context(), clientsets, CephClusterNamespace, OperatorNamespace, staleSubvol)
+	},
+}
+
+var deleteCmd = &cobra.Command{
+	Use:                "delete",
+	Short:              "Deletes the stale subvolume.",
+	DisableFlagParsing: true,
+	Args:               cobra.MaximumNArgs(3),
+	Run: func(cmd *cobra.Command, args []string) {
+		clientsets := GetClientsets(cmd.Context())
+		VerifyOperatorPodIsRunning(cmd.Context(), clientsets, OperatorNamespace, CephClusterNamespace)
+		subList := args[0]
+		fs := args[1]
+		svg := args[2]
```

Comment:
Need some validation that there are three args or else it crashes:
```
% bin/kubectl-rook-ceph subvolume delete      
Warning: rook version 'rook: v1.13.0-alpha.0.5.gd42c498ea-dirty' is running a pre-release version of Rook.

panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/rook/kubectl-rook-ceph/cmd/commands.glob..func17(0x106eb35e0, {0x1400003a0e0, 0x0, 0x0?})
	/Users/tnielsen/src/go/src/github.com/rook/kubectl-rook-ceph/cmd/commands/subvolume.go:49 +0x118
github.com/spf13/cobra.(*Command).execute(0x106eb35e0, {0x1400003a0e0, 0x0, 0x0})
	/Users/tnielsen/src/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0x7cc
github.com/spf13/cobra.(*Command).ExecuteC(0x106eb2a60)
	/Users/tnielsen/src/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x380
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/tnielsen/src/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
main.main()
	/Users/tnielsen/src/go/src/github.com/rook/kubectl-rook-ceph/cmd/main.go:25 +0x100
```

Reply:
might want to use `Args: cobra.ExactArgs(3)` here.

Reply:
sure will add it.